policy/agent.go

```diff
@@ -68,6 +68,7 @@ func (o *Agent) GetBackendName() string {
 // overwrite the result status or any of the values in the result trust vector.
 func (o *Agent) Evaluate(
 	ctx context.Context,
+	appraisalContext map[string]interface{},
```

Comment:
This change also means that the comment on Evaluate at line 66 needs to change!

Reply:
I don't believe so? The comment is no less valid than before. This new parameter doesn't change what the function is doing, or how it is doing it. This new parameter doesn't bear mentioning any more than the existing `ctx`, `submod`, `appraisal`, etc.